src/app.js

```diff
@@ -415,6 +400,34 @@ var run = function () {
 
   $('.asmOutput button').click(function () { $(this).parent().find('pre').toggle(); });
 
+  // ----------------- version selector-------------
+
+  // var soljsonSources is provided by bin/list.js
+
+  var versionList = true;
+  try {
+    soljsonSources;
```

Comment:
`window.soljsonSource === undefined`? (or document, whatever the global context is)